scripts/analytics/run_keen_summaries.py

```diff
@@ -8,6 +9,10 @@
 from scripts.analytics.preprint_summary import PreprintSummary
 from scripts.analytics.institution_summary import InstitutionSummary
 from scripts.analytics.base import DateAnalyticsHarness
+from scripts.utils import add_file_logger
+
+logger = logging.getLogger('scripts.analytics')
+logging.basicConfig(level=logging.INFO)
```

Comment:
Why was this added?

Reply:
I suppose the log level isn't needed, actually, but I added the file logger in these to address the comment "the file logger should be created in the run_main function for the scripts being run," and I needed a logger to pass into the individual summary scripts that were being run by celery

Reply:
I was only referring to the `basicConfig` call. Can it be removed?

Reply:
Ah ok! Yes and removed!